src/Templating/Helper/BlockHelper.php

```diff
@@ -385,7 +385,7 @@ public function exists(string $type): bool
      * @param string|array<string, mixed>|BlockInterface $block
      * @param array<string, mixed>                       $options
      */
-    public function render($block, array $options = []): string
+    public function render(string|array|BlockInterface $block, array $options = []): string
```

Comment:
not sure about this one, class is not final

Reply:
Indeed

Reply:
it is sad having to disable a rule for this edge cases, there are not a lot of places where we care about of this.

Reply:
I've opened https://github.com/rectorphp/rector/issues/7823 but got no response so far.

But I'm fine disabling this rule so far.

Reply:
I can do it tomorrow, anyway it would be nice to take the time and try the rule on the remaining repositories, to see if some of the changes are BC and apply them right away.

Reply:
There is not a lot of remaining repository https://github.com/pulls?q=is%3Aopen+is%3Apr+author%3ASonataCI+archived%3Afalse :)